src/controllers/users.js

```diff
@@ -23,7 +23,17 @@ export default class usersController {
       }, process.env.SECRETKEY);
       res.status(201).json({ message: 'user successfully created', token });
     } catch (error) {
-      res.status(400).json({ error });
+      if (`${error.errors[0].message}` === 'passport must be unique') {
```

Comment:
If you can find another way to check this error without equalising with long string like that it would be better.

Reply:
@EugeneMunya  Here is a proposed solution

<img width="832" alt="Screen Shot 2020-02-12 at 16 43 51" src="https://user-images.githubusercontent.com/48021034/74345578-f8b1a600-4db6-11ea-9bf2-8f44cbea637b.png">